staff/david-cano/frontend/arroz/Arroz.prototype.shift.js

```diff
@@ -0,0 +1,13 @@
+//El método shift() elimina el primer elemento del array y lo retorna. Este método modifica la longitud del array.
+
+Arroz.prototype.shift = function () {
+    var deletedElement2 = this[0];
+
+    for (var i = 0; i < this.length; i++) {
```

Comment:
review this logic, look the delete method, and maybe the for bucle is not necesary